src/main.ts

```diff
@@ -314,6 +315,13 @@ async function run(): Promise<void> {
       missing: [...missingSnapshots],
       added: [...newSnapshots],
     };
+    // This allows using Discover to distinguish transactions that require approval
+    const {changed, missing} = results;
+    if (changed.length + missing.length > 0) {
```

Comment:
This code is based on this logic:
https://github.com/getsentry/action-visual-snapshot/blob/766ee9288e2c8611377e678ed6c5742ca6b8d52b/src/api/finishBuild.ts#L52-L55

Reply:
Can you add a comment there too, that this needs to be kept in sync? You could also create a function that takes BuildResults and returns the action string so that it's easier to keep in sync

Reply:
Or instead of having this logic here, you could have 3 tags for the length of changed/missing/added - that way you can handle the logic at query time.